Core/clim-core/dialog-views.lisp

```diff
@@ -172,6 +172,7 @@
 (define-presentation-method accept-present-default
     ((type completion) stream (view option-pane-view) default default-supplied-p
      present-p query-identifier)
+  (declare (special *accepting-values-stream*))
```

Comment:
Shouldn't this be a globally declared `DEFVAR` instead?

Reply:
Semantically, yes. Practically, does it make a difference?

Reply:
yes, people usually look for where the variable is defined, if it is only declared then it may be confusing..